test/close-request.test.js

```diff
@@ -0,0 +1,28 @@
+import t from 'tap'
+import Fastify from 'fastify'
+import FakeTimers from '@sinonjs/fake-timers'
+import slowDownPlugin from '../index.js'
+import { HEADERS } from '../lib/constants.js'
+
+t.test('Check the response time with 2 enpoints', async t => {
+  t.context.clock = FakeTimers.install()
+  const fastify = Fastify()
+  await fastify.register(slowDownPlugin)
+  fastify.get('/cancel', async () => 'Hello from Fastify!')
```

Comment:
if you have an endpoint called cancel, just respond with something like 'cancel' so it's more obvious what response it is